### DISCUSSION TITLE
could add a LATAM page

### DISCUSSION THREAD

**Discussion by DekuQJ03** [anidl/multi-downloader-nx#196] [Ideas]
if it were possible I would like them to add the page animeonegai. com , this is only from Latin America and would be very useful for us to download dubbing, I hope you take it into account

**Comment by izu-co** [comment]
I don't think it is possible for me for the following reasons:
 - I don't speak the language and had to translate everything 
 - It is `A LOT` of work to integrate a new service and would require a ton of time, I simply don't have
 
If someone from the LATAM community is willing to lay the groundwork, however, I would be willing to `help` with the integration.
